hw03_rileyj3.py

```diff
@@ -0,0 +1,111 @@
+######################################################################
+# Author: Bryce Riley
+# Username: rileyj3
+#
+# Assignment: HW03 : Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: To create something complex and learn about github
+# Link to Document: https://docs.google.com/document/d/1EzPApStfIckqskXxnh8CNhMuNgis_GTE2FZZHHZvji4/edit?usp=sharing
+######################################################################
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor('turquoise')
+
+bryce = turtle.Turtle()
+
+
+def build_house():
+    """
+    Have turtle build me a nice house.
+    """
+    pass
```

Comment:
Help text from the template. Remove.

Reply:
I'm not too sure what you mean by this professor. Are you talking about my docstring?

Reply:
I'm talking about the `pass` line. These comments always refer to the line of code just above the comment.